File: back/src/main/java/br/com/Helppy/model/Usuario.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package br.com.Helppy.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import lombok.experimental.SuperBuilder;

/**
 *
 * @author criad
 */
@Data
@Setter
@Getter
@ToString
@SuperBuilder
@AllArgsConstructor
@NoArgsConstructor
public class Usuario {
   protected int idUsuario;
   protected String nome;
   protected String email;
   protected String senha;
   protected int acesso;
}

File: front/src/app/model/usuario.ts
export class Usuario {
  idUsuario: number;
  nome: string;
  email: string;
  senha: string;
  acesso: number;

  constructor() {
    this.idUsuario = 0;
    this.nome = '';
    this.email = '';
    this.senha = '';
    this.acesso = 0;
  }
}
